build.gradle

```diff
@@ -39,6 +39,18 @@ ext {
     generatedMetamodels = "build/generated/sources/annotationProcessor/java/main/org/candlepin/subscriptions"
 }
 
+spotless {
+    java {
+        target project.fileTree(project.rootDir) {
+            include '**/*.java'
+            exclude '**/build/**'
+        }
+        enforceCheck false
+        googleJavaFormat()
+        licenseHeaderFile "${projectDir}/config/codestyle/HEADER.txt"
+        toggleOffOn()
```

Comment:
This enables a cool feature that lets you specify code that you don't want to violate rules or be reformatted

Read more here: https://github.com/diffplug/spotless/tree/main/plugin-gradle#spotlessoff-and-spotlesson

Reply:
Same deal, wanna put these useful details in comments in `build.gradle`?